### ISSUE TITLE
set ROUGE-1.5.5.pl and RELEASE-1.5.5/data

### ISSUE THREAD

**Issue by MahdiDoost** [tagucci/pythonrouge#23]
In the name of Allah
Hello,

> The only things you need to evaluate ROUGE score is to specify the paths of ROUGE-1.5.5.pl and RELEASE-1.5.5/data in this package.

In new version how do I set ROUGE-1.5.5.pl and RELEASE-1.5.5/data?
Thank you

**Comment by MahdiDoost** [comment]
In the name of Allah
Hello,

> In new version how do I set ROUGE-1.5.5.pl and RELEASE-1.5.5/data?


I use global member to this.
````
rouge = Pythonrouge(summary_file_exist=False
                            , summary=summary
                            , reference=reference
                            , recall_only=True      # If you need only recall of ROUGE metrics, set recall_only=True
                            , n_gram=n_gram
                            , ROUGE_SU4=False
                            , ROUGE_L=False
                            , stemming=stemming
                            , stopwords=stopwords
                            , word_level=True
                            , length_limit=True
                            , length=250
                            , use_cf=False
                            , cf=95
                            , scoring_formula="average"
                            , resampling=True
                            , samples=1000
                            , favor=True
                            , p=0.5)
rouge.ROUGE_path='./virSumFarsi/lib/python3.5/site-packages/pythonrouge/RELEASE-1.5.5/ROUGE-1.5.5.pl'

rouge.data_path=./virSumFarsi/lib/python3.5/site-packages/pythonrouge/RELEASE-1.5.5/data/'

score = rouge.calc_score()
```
`

**Comment by tagucci** [comment]
In new version of pythonrouge, you do not need to set ROUGE path.
pythonrouge automatically set ROUGE path:)

**Comment by MahdiDoost** [comment]
In the name of GOD
Hello
Thank you for answer.
> In new version of pythonrouge, you do not need to set ROUGE path.
pythonrouge automatically sets ROUGE path:)


I set ROUGE_path and data_path and get below result:

`{'ROUGE-2': 0.25, 'ROUGE-1': 0.42857}`

and when I don 't set ROUGE_path and data_path and get below result:
```
Traceback (most recent call last):
  File "/home/aliasghar/sumFarsi/prjSumFarsi/Rouge.py", line 200, in <module>
    print(rouge.setting_score_rouge_sentence(summary,reference,2,False,False))
  File "/home/aliasghar/sumFarsi/prjSumFarsi/Rouge.py", line 138, in setting_score_rouge_sentence
    score = rouge.calc_score()
  File "/usr/local/lib/python3.5/dist-packages/pythonrouge/pythonrouge.py", line 335, in calc_score
    output = subprocess.check_output(rouge_cmd, stderr=subprocess.STDOUT)
  File "/usr/lib/python3.5/subprocess.py", line 316, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 398, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['perl', '/usr/local/lib/python3.5/dist-packages/pythonrouge/RELEASE-1.5.5/ROUGE-1.5.5.pl', '-e', '/usr/local/lib/python3.5/dist-packages/pythonrouge/RELEASE-1.5.5/data', '-a', '-n', '2', '-x', '-l', '250', '-f', 'A', '-r', '1000', '-p', '0.5', './tmpjvx9v0w3/setting.xml']' returned non-zero exit status 255
```

**Comment by tagucci** [comment]
Did you update pythonrouge?
If not, try blow command.
```
pip uninstall pythonrouge
pip install git+https://github.com/tagucci/pythonrouge.git
```

**Comment by MahdiDoost** [comment]
Yes, I update pythonrouge.

**Comment by tagucci** [comment]
OK, so can you try below?
```
cd /usr/local/lib/python3.5/dist-packages/pythonrouge/RELEASE-1.5.5/data/
rm WordNet-2.0.exc.db
./WordNet-2.0-Exceptions/buildExeptionDB.pl ./WordNet-2.0-Exceptions ./smart_common_words.txt ./WordNet-2.0.exc.db
```

**Comment by MahdiDoost** [comment]
I try to run your code and get  Permission denied.
```
$ rm WordNet-2.0.exc.db
rm: remove write-protected regular file 'WordNet-2.0.exc.db'? y
rm: cannot remove 'WordNet-2.0.exc.db': Permission denied

```

**Comment by tagucci** [comment]
so, try below
```
sudo rm WordNet-2.0.exc.db
sudo ./WordNet-2.0-Exceptions/buildExeptionDB.pl ./WordNet-2.0-Exceptions ./smart_common_words.txt ./WordNet-2.0.exc.db
```

**Comment by MahdiDoost** [comment]
I get error:
```
Traceback (most recent call last):
  File "/home/aliasghar/sumFarsi/prjSumFarsi/Rouge.py", line 201, in <module>
    print(rouge.setting_score_rouge_sentence(summary,reference,2,False,False))
  File "/home/aliasghar/sumFarsi/prjSumFarsi/Rouge.py", line 137, in setting_score_rouge_sentence
    score = rouge.calc_score()
  File "/usr/local/lib/python3.5/dist-packages/pythonrouge/pythonrouge.py", line 335, in calc_score
    output = subprocess.check_output(rouge_cmd, stderr=subprocess.STDOUT)
  File "/usr/lib/python3.5/subprocess.py", line 316, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 398, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['perl', '/usr/local/lib/python3.5/dist-packages/pythonrouge/RELEASE-1.5.5/ROUGE-1.5.5.pl', '-e', '/usr/local/lib/python3.5/dist-packages/pythonrouge/RELEASE-1.5.5/data', '-a', '-n', '2', '-x', '-l', '250', '-f', 'A', '-r', '1000', '-p', '0.5', './tmpzl78ah1g/setting.xml']' returned non-zero exit status 13
```

**Comment by vintersnow** [comment]
FYI. I had a similar error (`non-zero exit status 2`)
After I run `cpan XML::DOM` and rebuild WordNet-2.0.exc.db, it was fixed (#11)

**Comment by yucoian** [comment]
> In new version of pythonrouge, you do not need to set ROUGE path.
> pythonrouge automatically sets ROUGE path:)

HOWEVER, you never update the README file.....
